tox.ini

```diff
@@ -44,6 +44,7 @@ install_command =
 passenv =
   CURL_CA_BUNDLE  # https proxies, https://github.com/tox-dev/tox/issues/1437
   HOME
+  PYTEST_*  # allows developer to define their own preferences
```

Comment:
This one is controversial. Using posargs already allows to override the args, what else do you need?

Reply:
Is not controversial at all, it allows developer to configure set of persistent personal preferences for running pytest, and avoid having to pass extra options each time you call pytest.

For example, my local ones look like ``export PYTEST_ADDOPTS='-s -n0 --ff --verbosity=0'``

Reply:
This sounds like it'll allow people to accidentally break our setup and complain about it being broken...

Reply:
Should we stop selling cars because some people may not know how to drive them? If you are really afraid of this, you can make a PR that adds a warning message when tests failed and PYTEST_ADDOPTS was defined. Still, I doubt that this is a real risk. There are lots of other stuff that can produce failures and unless we see a clear pattern (multiple bugs reported), I doubt we should invest in them.